src/app/home/avatar-generator/avatar-types.ts.ts

```diff
@@ -0,0 +1,11 @@
+export type KeyAvatar = 'hair' | 'clothes' | 'mouth' | 'earrings' | 'eyebrows' | 'eyes' | 'glasses' | 'body' | 'hat' | 'beard';
+
```

Comment:
Just in case, this file has double ".ts" extension

Reply:
🙈